components/brave_shields/browser/ad_block_subscription_service_manager.cc

```diff
@@ -60,7 +60,22 @@ bool ParseOptionalStringField(const base::Value* value,
   }
 }
 
-const base::TimeDelta kListUpdateInterval = base::Days(7);
+bool ParseExpiresWithFallback(const base::Value* value, uint16_t* field) {
+  if (value == nullptr) {
+    *field = kSubscriptionDefaultExpiresHours;
+    return true;
+  } else if (!value->is_int()) {
+    return false;
+  } else {
+    int64_t i = value->GetInt();
+    if (i < 0 || i > 14 * 24) {
```

Comment:
lets make another const or use `2 * kSubscriptionDefaultExpiresHours`

Reply:
I'll make another const here; it's just a "maximum valid" value which doesn't necessarily have anything to do with the default

Reply:
added `kSubscriptionMaxExpiresHours`